main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -33,13 +33,41 @@ object CompleteProvider {
     * Returns a list of auto-complete suggestions.
     */
   def autoComplete(uri: String, pos: Position, line: Option[String], word: Option[String])(implicit index: Index, root: TypedAst.Root): Iterable[CompletionItem] = {
-    // Ordered by priority.
-    getVarSuggestions(uri, pos, line, word) ++
-      getDefAndSigSuggestions(uri, pos, line, word) ++
-      getInstanceSuggestions(uri, pos, line, word) ++
-      getWithSuggestions(uri, pos, line, word) ++
-      getKeywordCompletionItems(line, word) ++
-      getSnippetCompletionItems(line, word)
+    word match {
+      case None => Nil;
+      case Some(w) => 
+        if (w.startsWith("@")) {
```

Comment:
This seems reasonable, but I would move the startsWith check into getAnnotationCompletionItems

Reply:
Ah- on second thought, Its because you don't want to give the other suggestions when writing `@...` Hmm.

Reply:
Perhaps we need a helper function? `isAnnotationPrefix` or whatever. Not sure.

Reply:
Yeah, you understand my thinking here. Happy to structure it however you see fit.